### PR TITLE
Refer to table-scan-node-partitioning-min-bucket-to-task-ratio in doc

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: docs/src/main/sphinx/admin/properties-optimizer.rst [hunk 1]
```diff
@@ -125,7 +125,9 @@ join output rows can be skipped.
 
 Use connector provided table node partitioning when reading tables.
 For example, table node partitioning corresponds to Hive table buckets.
-When set to ``true`` each table partition is read by a separate worker.
+When set to ``true`` and minimal bucket to task ratio, set
+in ``preferred-write-partitioning-min-number-of-partitions`` is matched or exceeded,
```
Comment (findepi) [inline docs/src/main/sphinx/admin/properties-optimizer.rst:129]: the comma should be after property name, before "is"

Comment (sopel39) [reply docs/src/main/sphinx/admin/properties-optimizer.rst:129]: this is based on `docs/src/main/sphinx/admin/properties-writer-scaling.rst:37`

Comment (mosabua) [reply docs/src/main/sphinx/admin/properties-optimizer.rst:129]: Use something like this to make it simpler to understand:

```
When set to ``true`` and minimal partition to task ratio is matched or exceeded, each table partition is read by a separate worker. The minimal ratio is defined in ``preferred-write-partitioning-min-number-of-partitions``.
```

**Review by mosabua — CHANGES_REQUESTED** [review]
I would break it up into two sentences. And maybe even end the paragraph after.

**Comment by sopel39** [comment]
> I would break it up into two sentences. And maybe even end the paragraph after.

done. I'm keeping `table-scan-node-partitioning-min-bucket-to-task-ratio` for now

**Review by mosabua — APPROVED** [review]
lgtm .. note that the property name uses bucket .. but in docs here we only use partition at this stage.

**Review by findepi — APPROVED** [review]

**Pull request merged**